magento/magento2
# M2 2266

### Description
This PR fixes magento/magento2#2266 where the actions dropdown menu is not aligned properly in the admin grids when the action column is not the last column.

### Fixed Issue
1. magento/magento2#2266: Action column menu does not stay over the action column


### Manual testing scenarios
1. Open the admin panel.
2. Go to the cms pages list: Content->Pages (from the main menu)
3. After the grid is rendered add this code in the file `app/code/Magento/Cms/view/adminhtml/ui_component/cms_page_listing.xml` (I edited that file to make it easier to explain. It also happens if I add a new column using the file cms_page_listing.xml from my module also)
```
        <column name="dummy">
            <settings>
                <filter>text</filter>
                <editor>
                    <validation>
                        <rule name="validate-identifier" xsi:type="boolean">true</rule>
                    </validation>
                    <editorType>text</editorType>
                </editor>
                <label translate="true">Dummy</label>
            </settings>
        </column>
```
4.Refresh the pages list.
5. click on the actions link for the first item in the grid.

Expected result: 
  - The actions menu is over the actions column  
![](http://i.imgur.com/QknMf3n.png)

Actual Result  
 - The actions menu is over the last column.  
![](https://cloud.githubusercontent.com/assets/2447784/16612611/78b0fe6e-4370-11e6-922b-e72c7edc231b.png)

### Contribution checklist
